src/main/java/kpl/fiml/project/presentation/ProjectController.java

```diff
@@ -65,41 +112,91 @@ public ResponseEntity<Void> updateRewards(@PathVariable("projectId") Long projec
         return ResponseEntity.ok().build();
     }
 
+    @Operation(summary = "프로젝트 최종 업로드", description = "프로젝트 정보 모두 입력 후 최종 업로드")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "프로젝트 최종 업로드 성공"),
+            @ApiResponse(responseCode = "400 - 1", description = "프로젝트 정보 모두 입력하지 않음"),
+            @ApiResponse(responseCode = "400 - 2", description = "프로젝트 업로드 가능 상태가 아님"),
+            @ApiResponse(responseCode = "400 - 3", description = "프로젝트 펀딩 기간이 올바르지 않음"),
```

Comment:
이거 야매일 수도 있지만 전 이런 식으로 해결했었습니다...
```suggestion
            @ApiResponse(responseCode = "400 - 3", description = "프로젝트 펀딩 기간이 올바르지 않음 \n 또다른 400 에러 발생 원인 \n 또다른 400 에러 발생 원인2"),
```

Reply:
수정 완료~